packages/utils/src/verification/contactRequest.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  MAX_CONTACT_REQUEST_TITLE_LENGTH,
+  MAX_CONTACT_REQUEST_MESSAGE_LENGTH,
+  MIN_CONTACT_REQUEST_TITLE_LENGTH,
+  MIN_CONTACT_REQUEST_MESSAGE_LENGTH,
+  MIN_NAME_LENGTH,
+  MAX_NAME_LENGTH,
+  ALPHABETIC,
+} from "@/constants";
+import { FieldError } from "@litespace/types";
+
+export function isValidName(
+  name: string
```

Comment:
The params should be of type `unkown`. 

```suggestion
  name: unkown
```

Reply:
I don't get it. Why it should be unknown?!